src/test/suite/fixtures/recorded/languages/php/chuckValue2.yml

```diff
@@ -0,0 +1,29 @@
+languageId: php
+command:
+  version: 1
+  spokenForm: chuck value
+  action: remove
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: value, includeSiblings: false}
+initialState:
+  documentContents: |-
+    <?php
+
+    $value = 'Hello world';
+  selections:
+    - anchor: {line: 2, character: 7}
+      active: {line: 2, character: 7}
+  marks: {}
+finalState:
+  documentContents: |-
+    <?php
+
+    $value =;
```

Comment:
Should probably remove `=`? Idk if that's not syntactically valid then not sure there's much benefit

Reply:
Yeah good call 👍